.eleventy.js

```diff
@@ -150,13 +155,14 @@ module.exports = function(config) {
   // FILTERS
   //----------------------------------------------------------------------------
   config.addFilter('findBySlug', findBySlug);
+  config.addFilter('findTags', findTags);
```

Comment:
What do you think about renaming this file to `validateTags` or `allowedTags`? Just trying to think of some term that's very descriptive of what it's doing. It's kind of a filter step